shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -1101,10 +1102,7 @@ stakeDistr u ds ps = (Stake $ Map.keysSet activeDelegs ◁ stake, delegs)
       DState (StakeKeys stkeys) rewards' delegs ptrs' _ _ = ds
       PState (StakePools stpools) _ _ _                   = ps
       outs = consolidate u
-      stake = baseStake' `Map.union` pointerStake `Map.union` rewardStake'
-      Stake baseStake'   = baseStake outs
-      Stake pointerStake = ptrStake outs ptrs'
-      Stake rewardStake' = rewardStake rewards'
+      Stake stake = baseStake outs ⊎ ptrStake outs ptrs' ⊎ rewardStake rewards'
```

Comment:
would it make sense to actually create the `aggregate_+` function explicitly? That would help converging the formal and executable spec. There is also a related issue, cf. #741

Reply:
yea, we could do that. I think the awkwardness here in general comes for the fact that we do not really using relations, but instead maps.

the `stakeRelation` here is a relation, but if everything is being implemented as a map then we have to be careful (ie use union-override-plus) when taking the unions:

![stakeRelation](https://user-images.githubusercontent.com/943479/63590193-242aa500-c579-11e9-9ccd-40a990352a76.png)